howso/react_aggregate.amlg

```diff
@@ -677,41 +767,40 @@
 		(if (= conditioned_residuals (true))
 			(let
 				(assoc
-					action_condition_precision (if
-						(= (get details "action_condition_precision") (null) )
-						"exact"
-						(get details "action_condition_precision")
-					)
-					context_condition_precision (if
-						(= (get details "action_condition_precision") (null) )
-						"exact"
-						(get details "action_condition_precision")
-					)
+					action_condition_precision
+						(if (= (get details "action_condition_precision") (null) )
+							"exact"
+							(get details "action_condition_precision")
+						)
```

Comment:
`(or (get details "action_condition_precision") "exact")`

I believe this should work the same way, if so do this for the spot below as well